cisco_aci/assets/monitors/critical_health_score.json

```diff
@@ -0,0 +1,27 @@
+{
+	"version": 2,
+	"created_at": "2024-06-21",
+	"last_updated_at": "2024-06-21",
+	"title": "Health score of device is critical",
+	"description": "A health score is a heuristic generated by Cisco ACI to represent the state of a leaf or spine node and represents if the device is good shape. The alert will trigger if the health is in critical state.",
+	"definition": {
+		"name": "[Cisco ACI] Health score of device {{node_id.name}} is critical",
+		"type": "query alert",
+		"query": "avg(last_5m):avg:cisco_aci.fabric.node.health.cur{*} by {fabric_pod_id,node_id,apic_role} < 50",
+		"message": "A health score is a heuristic generated by Cisco ACI to represent the state of a leaf or spine node. It provides a high level understanding if something isn't in a good shape. For example, a spine with 6 power supplies that has 1 power supply failure isn't going to lower it's health score as much as a leaf switch with two power supplies that loses one.\n\n{{#is_alert}}\n{{node_id.name}} in namespace {{device_namespace.name}} is reporting low health score (under 50%).\n{{/is_alert}}\n\n{{#is_warning}}\n{{node_id.name}} in namespace {{device_namespace.name}} is reporting lower health score (under 90%).\n{{/is_warning}} \n\n{{#is_recovery}}\n{{node_id.name}} in namespace {{device_namespace.name}} is reporting healthy score (over 90%).\n{{/is_recovery}}",
+		"tags": [],
+		"options": {
+			"thresholds": {
+				"critical": 50,
+				"warning": 90
```

Comment:
How did you come up with 50/90? Quickly glimpsing online, Cisco has a page that recommends setting an alert on 90%. Would 50% be too low?

Reply:
i think i found what you're [referring](https://documentation.solarwinds.com/en/success_center/npm/content/npm_aci_alerts_and_reports.htm#:~:text=Alert%20on%20health%20score%20values,want%20to%20be%20alerted%20about.) to which is for the ACI tenant (this would be for the node / device itself)

which i thought i found a doc but i should've saved it all i can find is this [anymore](https://documentation.solarwinds.com/en/success_center/npm/content/npm-monitor-cisco-aci.htm)  for ACI members